frontend/src/components/dashboard/home/ActiveVisitorsCard.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+    <div class="card bg-white visitors-card">
+        <div class="d-flex justify-content-between align-items-center card-top-row">
+            <p class="card-text mb-0">
+                Active users
+            </p>
+            <p class="card-count text-right mb-0">
+                {{ activeUsersCount }}
+            </p>
+        </div>
+        <div class="d-flex justify-content-between align-items-center card-top-row">
+            <p class="card-text mb-0">
+                Page views now
+            </p>
+            <p class="card-count text-right mb-0">
+                {{ pageViewsCount }}
+            </p>
+        </div>
+        <p
+            class="card-title"
+        >
+            Top active pages
+        </p>
+        <ul
+            class="links-list m-0 p-0"
+        >
+            <li
+                v-for="page in topPages"
+                :key="page"
+            >
+                <a
+                    class="link-item"
+                    href="#"
+                >
+                    {{ page }}
+                </a>
+            </li>
+        </ul>
+        <div
+            class="text-center"
+        >
+            <a
+                href="#"
+                class="btn card-button font-weight-light"
+            >
+                Real time
+            </a>
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'ActiveVisitorsCard',
+        props: {
+            activeUsersCount: {
+                type: Number,
+                default: 250
+            },
+            pageViewsCount: {
+                type: Number,
+                default: 31
+            },
+            topPages: {
+                type: Array,
+                default: function () {
```

Comment:
Yes, it is fake data. And default values ​​of type array or object in props should use function. 
This will be changed later.

Reply:
I meant to use arrow function instead of this old `function`)

Reply:
Like `default() {}`

Reply:
ok. It is also possible `default: () => {}` ?

Reply:
should be, I usually use `method() {}` instead of `method: () => {}` since in first variant it's method definition inside object, and second variant - assign function to object key

Reply:
ok, thanks.